### DISCUSSION TITLE
custom chatterino badges for streamer

### DISCUSSION THREAD

**Discussion by Seemsyyy** [Chatterino/chatterino2#2832] [Ideas] [answered]
Apu

**Comment by zneix** [comment, marked as answer]
tl;dr This issue is out of scope of the Chatterino project.

Chatterino is meant to be a simple chat client. Supporter badges mostly exist just so some kind generous users can donate if they want to to support the project, show their appreciation and get something in return.

I don't see any particular reason to now work on a "shop" for micro-transactions. That would be only time consuming, could cause many issues with people not trusting us with the payments, potential "customer service" and god knows what else.

As of time of writing this comment, we've got over 300 issues and 25 pull requests to go through and work on. It's straight up impossible to also add this whole system which I personally think would be very niche and may make people think that a simple chat app is "pay to win" and bloated.
